openshift-storage-libs/openshiftstoragelibs/baseclass.py

```diff
@@ -1097,3 +1098,38 @@ def verify_if_more_than_n_percentage_pod_restarted(
             msg = "Out of {} pods {} pods restarted {}".format(
                 len(pods_new), len(pods_restart), pods_restart)
             raise AssertionError(msg)
+
+    def validate_glusterfsd_pod_memory(self, size_limit):
+        """Validate memory usage of glusterfsd process.
+
+        Args:
+            size_limit (int): Expected memeory usage.
+
+        """
+        # Cmd to fetch the pid and memory usage
+        get_glusterds_pid = "pgrep glusterfsd"
```

Comment:
Change this variable name to: 
get_glusterfsd_pid

Reply:
Done